src/api/spec/jobs/update_released_binaries_job_spec.rb

```diff
@@ -14,7 +14,7 @@
   let!(:repository) { create(:repository, project: user.home_project) }
 
   describe '.update_binary_releases' do
-    subject { described_class.update_binary_releases(repository, 'foo') }
+    subject { described_class.new.update_binary_releases(repository, 'foo') }
```

Comment:
`update_binary_releases` is a private instance method. It should have been called with `described_class.new.send(...)`, like `update_binary_releases_via_json`.

But `update_binary_releases` gets removed with a refactoring. This can stay as it is. No issue here.